client/src/app/site/pages/meetings/pages/motions/pages/motion-export/components/motion-export/motion-export.component.ts

```diff
@@ -257,19 +260,28 @@ export class MotionExportComponent extends BaseComponent implements AfterViewIni
         }
     }
 
+    public override ngOnDestroy(): void {
+        this.storeService.set(`motion-export-selection`, this.dialogForm.value);
+        super.ngOnDestroy();
+    }
+
     // React to changes on the file format
     public tabChanged = (tabChangeEvent: MatTabChangeEvent): void => {
         this.isCSVExport = this.fileFormats[tabChangeEvent.index] === ExportFileFormat.CSV;
         this.isXLSXExport = this.fileFormats[tabChangeEvent.index] === ExportFileFormat.XLSX;
     };
 
     public animationDone(): void {
```

Comment:
```suggestion
    public afterTabChanged(): void {
```

This was the confusing part. Please change before merging.

Reply:
Renaming: Done.